test/manual-test-examples/p5.Image/GIF/sketch.js

```diff
@@ -0,0 +1,42 @@
+let img0, img1;
+
+let layer0, layer1;
+
+function preload() {
+  img0 = loadImage('./assets/counting-gif@endlessLoop-1secDelay.gif');
+  img1 = loadImage('./assets/counting-gif@1xLoop-0.2secDelay.gif');
```

Comment:
this it nitpicky but can you rename these files to something a little simpler?

Reply:
disregard my last comment, just realized this is a test not a reference example!